### DISCUSSION TITLE
Your module is completely miraculous to me after Android 12.

### DISCUSSION THREAD

**Discussion by Elaman0117** [luisbocanegra/substratum-tweaks#9] [General]
I really don't like the "Material You" design, and this is the universal solution I use to make my Android 13 devices look like Android 11. It should be very popular, but it took me a long time to find it... Have you tried promoting it in some forums? I really like this module and hope to keep developing it.

**Comment by luisbocanegra** [comment]
> I really don't like the "Material You" design, and this is the universal solution I use to make my Android 13 devices look like Android 11. 

Thanks, glad this has been useful to you, to me is still one of the first things I install if ever have to reset my small phone (5.5 inch) or switch roms to make it usable again :smile: 
 
> It should be very popular, but it took me a long time to find it... Have you tried promoting it in some forums? 

Yeah... I didn't, just some mentions of it on reddit threads :eyes:. maybe promoting it a little or even publishing it on android store could still be a good idea?...

> I really like this module and hope to keep developing it.

I will, still open to suggestions of things that **can** be added and pull requests are welcomed, it's the main reason this project is open source after all.

**Reply by Elaman0117** [reply]
>
> Thanks, glad this has been useful to you, to me is still one of the first things I install if ever have to reset my small phone (5.5 inch) or switch roms to make it usable again 😄
>
Is it a Pixel 3? If so then what a coincidence, I used it from high school to freshman year.
> Yeah... I didn't, just some mentions of it on reddit threads 👀. maybe promoting it a little or even publishing it on android store could still be a good idea?...
>
Yep, that's the path I got here...  I think this software definitely has a lot of room for development, but the people who are most likely to need it are people like me who know a little but are not proficient in Android. Not that F-Droid third-party sources are bad, but those of us (or maybe at least me) really don't think to add third-party sources. Open a few posts on XDA, Reddit, put on xda, F-droid (first-party library), I think it can be a good publicity, after all, at least I found a lot of excellent software through these channels. Although it is said that "Good wine needs no bush.", I hope that it can be found by more people.
> I will, still open to suggestions of things that **can** be added and pull requests are welcomed, it's the main reason this project is open source after all.
>
I'll try, but I don't know much about Android development, Substratum themes, or even the format of GitHub feedback. 🫠

**Reply by luisbocanegra** [reply]
> Is it a Pixel 3? If so then what a coincidence, I used it from high school to freshman year.

Nah Xiaomi Redmi Note 4 lol :skull: 

> > Yeah... I didn't, just some mentions of it on reddit threads eyes. maybe promoting it a little or even publishing it on android store could still be a good idea?...
> 
> Yep, that's the path I got here... I think this software definitely has a lot of room for development, but the people who are most likely to need it are people like me who know a little but are not proficient in Android. Not that F-Droid third-party sources are bad, but those of us (or maybe at least me) really don't think to add third-party sources. Open a few posts on XDA, Reddit, put on xda, F-droid (first-party library), I think it can be a good publicity, after all, at least I found a lot of excellent software through these channels. Although it is said that "Good wine needs no bush.", I hope that it can be found by more people.

Thanks for your advice, will try doing some of these when I have some time :+1: 
> > I will, still open to suggestions of things that **can** be added and pull requests are welcomed, it's the main reason this project is open source after all.
> 
> I'll try, but I don't know much about Android development, Substratum themes, or even the format of GitHub feedback. 🫠

Don't worry, I don't know much about android development either :upside_down_face:, knowing how to build it is enough. For "theming" there is a [guide](https://github.com/substratum/template) and a telegram group t. me / ThemingHelp in case you're interested

Most of the Android and Systemui overlays are obtained from existing ones that I've found interesting/useful:
- https://github.com/aosp-mirror/platform_frameworks_base/tree/main/core/res/res/values
- https://github.com/aosp-mirror/platform_frameworks_base/tree/main/packages/SystemUI/res/values

As you can see I mostly do config related things and not actual styling (doesn't mean is not welcomed :smiley:) because my knowledge there is limited too haha.
